### PR TITLE
HSEARCH-2247

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://hibernate.atlassian.net/browse/HSEARCH-2247

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Supersedes #1090 which needed some adjustments after latest developments on master.

File: engine/src/main/java/org/hibernate/search/indexes/spi/IndexManagerType.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * Since Hibernate Search supports different types of indexing and query technologies,
+ * such as embedding Apache Lucene or remote via Elasticsearch, the type of
+ * such storage is represented by an instance implementing this interface.
+ * <p>
+ * Instances of implementations of this interface could be used as keys in a Map,
+ * so make sure to implement appropriate equals and hashCode functions.
+ * <p>
+ * Instances of implementations of this interface could be used as keys in a Map,
+ * so make sure to implement appropriate equals and hashCode functions.
+ * <p>
```
Comment (gsmet) [inline engine/src/main/java/org/hibernate/search/indexes/spi/IndexManagerType.java]: Duplicate text:
> + * Instances of implementations of this interface could be used as keys in a Map,
> + * so make sure to implement appropriate equals and hashCode functions.

**Comment by gsmet** [comment]
Apart from the duplicate text in the comment, looks good to me.

I let you fix the comment and merge it?

Thanks!

**Comment by gunnarmorling** [comment]
Addressed @gsmet's remark and merged. Thx!

**Pull request closed without merging**